File: Assignment_1/Assignment1_2.py
"""-------------------------------------------------------------------------
                            Assignment1_2                         
                    (Student name - Vaishali Jorwekar)
                    Python by Marvellous Infosystems
-------------------------------------------------------------------------
Problem statement :
               - This program checks if number is even or odd
-------------------------------------------------------------------------"""

#-------------------------------------------------------------------------
# This function checks for even or odd number
#-------------------------------------------------------------------------
def ChkNum(number):
    print("-----------------------------------")
    if number % 2 ==0:
        print("Output : ",number," is Even Number")
    else:
        print("Output : ",number," is Odd Number")
#-------------------------------------------------------------------------   
# This function calls ChkNum(num) function with one parameter
#-------------------------------------------------------------------------
def main():
    print("Enter the number:")
    num= int(input())
    ChkNum(num)
#-------------------------------------------------------------------------
# Entry point of program
#-------------------------------------------------------------------------
if __name__=="__main__":
    main()
#-------------------------------------------------------------------------    

File: JavaPractice/Assignment1_2.java
/*-------------------------------------------------------------------------
                            Assignment1_2                         
                    (Student name - Vaishali Jorwekar)
-------------------------------------------------------------------------
Problem statement :
               - This program checks if number is even or odd
-------------------------------------------------------------------------*/
import java.util.Scanner;
class Assignment1_2{

    public static void chcekNumber(int number){
        if(number%2==0)
            System.out.println(number+":is EVEN number");
        else
            System.out.println(number+" is ODD Number");

    }
    public static void main(String args[]){
        Scanner scanObj=new Scanner(System.in);
        System.out.print("Enter the number:");
        int number=scanObj.nextInt();
        System.out.println("Entered number is :"+number);
        scanObj.close();
        chcekNumber(number);
        
    }

}
